tooltester/src/test/java/io/dockstore/tooltester/S3ClientTest.java

```diff
@@ -29,13 +29,13 @@ public void getToolTesterLogsWithToolIdAndToolVersionName() {
     @Test
     public void convertToolIdToPartialKey() throws UnsupportedEncodingException {
         String toolId = "#workflow/github.com/ENCODE-DCC/pipeline-container/encode-mapping-cwl";
-        Assert.assertEquals("workflow/github.com/ENCODE-DCC/pipeline-container%2Fencode-mapping-cwl", S3Client.convertToolIdToPartialKey(toolId));
+        Assertions.assertEquals("workflow/github.com/ENCODE-DCC/pipeline-container%2Fencode-mapping-cwl", S3Client.convertToolIdToPartialKey(toolId));
```

Comment:
I like to static import `assertEquals` so you only have `assertEquals` rather than `Assertions.assertEquals`

Reply:
Thanks, I have now included the static imports.